paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -321,7 +323,7 @@ class PaymentSheet internal constructor(
 
     /** Configuration for [PaymentSheet] **/
     @Parcelize
-    data class Configuration @JvmOverloads constructor(
+    data class Configuration internal constructor(
```

Comment:
This is most of the big change here.

If I simply add the param to the existing constructor I'd have to mark the whole constructor with the experimental API, which we don't want!

This moves the primary constructor to internal, and the new public constructor just matches what we had before.